src/System.IO.FileSystem/tests/FileStream/IsAsync.cs

```diff
@@ -40,6 +40,7 @@ public void FileOptionsAsynchronousConstructorArg()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetFX defaults useAsync to false for the SafeFileHandle/FileAccess constructor which leads to an ArgumentException for an async handle. In netcoreapp this constructor was fixed to check the async status of the handle and use that for the useAsync value.")]
```

Comment:
Same here; should we open an issue as netfx-port-consider or if there is one already open should we include it in the comment?